tools/jdk/remote_java_tools_aliases.bzl

```diff
@@ -0,0 +1,38 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A collection of macros that wrap native Bazel rules and select the given target
+# from an external java tools repository based on the platform.
+
+def _get_args(target, attr, **kwargs):
+    workspace_target_dict = {
+        "//src/conditions:linux_x86_64": ["@remote_java_tools_linux//" + target],
+        "//src/conditions:linux_aarch64": ["@remote_java_tools_linux//" + target],
```

Comment:
We don't have a separate ijar build for Aarch64, do we? But if not, how does this work? An ijar binary built for x86_64 surely won't run there...

Reply:
As discussed: build a source archive and direct `//conditions:default` to that?

Reply:
Actually I realized we can re-use any of the already existing archives (I choose the linux one). The BUILD file in the remote repository returns the prebuilt ijar binary for the 3 known platforms and the `cc_binary` target for all the others (+ remote execution). All the other targets are deploy jars and the platform doesn't matter in their case.